### PR TITLE
gofumpt: Init at 2019-07-29

### PR DESCRIPTION
**Pull request by rvolosatovs**
###### Motivation for this change
New package

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build gofumpt

File: pkgs/development/tools/gofumpt/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  src = fetchFromGitHub {
+    owner = "mvdan";
+    repo = "gofumpt";
+    rev = "96300e3d49fbb3b7bc9c6dc74f8a5cc0ef46f84b";
+    sha256 = "169hwggbhlr6ga045d6sa7xsan3mnj19qbh63i3h4rynqlppzvpf";
+  };
+
+  modSha256 = "1g7dkl60zwlk4q2gwx2780xys8rf2c4kqyy8gr99s5y342wsbx2g";
+
+  meta = with lib; {
```
Comment (symphorien) [inline pkgs/development/tools/gofumpt/default.nix]: can you add a description ?

Comment (rvolosatovs) [reply pkgs/development/tools/gofumpt/default.nix]: done!

File: pkgs/development/tools/gofumpt/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "gofumpt-${version}";
```
Comment (alexarice) [inline pkgs/development/tools/gofumpt/default.nix]: Can you use `pname = "gofumpt";` here instead?

File: pkgs/development/tools/gofumpt/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  version = "2019-07-29";
+
+  src = fetchFromGitHub {
+    owner = "mvdan";
+    repo = "gofumpt";
```
Comment (alexarice) [inline pkgs/development/tools/gofumpt/default.nix]: `repo = pname;`

**Comment by rvolosatovs** [comment]
@alexarice done!

**Comment by rvolosatovs** [comment]
@GrahamcOfBorg build gofumpt

**Review by kalbasit — APPROVED** [review]

**Pull request merged**